editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5418,6 +5430,22 @@ CanvasItemEditor::CanvasItemEditor() {
 		theme_menu->set_item_checked(i, i == theme_preview);
 	}
 
+	preview3d_menu = memnew(PopupMenu);
+	preview3d_menu->connect("id_pressed", callable_mp(this, &CanvasItemEditor::_switch_3d_preview));
+	preview3d_menu->set_name("Preview3DMenu");
+	preview3d_menu->add_radio_check_item("None", 0);
+	preview3d_menu->add_radio_check_item("Viewport 1", 1);
+	preview3d_menu->add_radio_check_item("Viewport 2", 2);
+	preview3d_menu->add_radio_check_item("Viewport 3", 3);
+	preview3d_menu->add_radio_check_item("Viewport 4", 4);
```

Comment:
These names should be made translatable via `TTR()` or `TTRC()`.

Also, I'm not sure if it's worth it. `"Viewport N"` could be turned into something like `vformat(TTR("Viewport %d"), i)` so they're easier to translate.